libs/langchain/langchain/retrievers/dingo.py

```diff
@@ -0,0 +1,60 @@
+"""Dingo Retriever"""
```

Comment:
what does this add over the vectorstore? if it doesnt add anything, may be best to delete

Reply:
> what does this add over the vectorstore? if it doesnt add anything, may be best to delete

I hava Fixed， removed this description

Reply:
i think what harrison means is that you can already take any vector store and turn it into a retriever pretty easily 
```python
vectorstore = Dingo(...)
retriever = vectorstore.as_retriever()
```

and that retriever will under the hood use the vectorstore to perform similarity search. so probably no need to manually implement a DingoRetriever class as well, can delete

Reply:
> and that retriever will under the hood use the vectorstore to perform similarity search. so probably no need to manually implement a DingoRetriever class as well, can delete

Thanks for reminding me, I have deleted